src/Hl7.Fhir.R5.Core/ElementModel/PocoBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿/* 
+ * Copyright (c) 2018, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://github.com/FirelyTeam/firely-net-sdk/blob/master/LICENSE
+ */
+
+
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+
+namespace Hl7.Fhir.ElementModel
+{
+    public static class PocoBuilderExtensions
```

Comment:
Why is this shim only present in the R5 project and not in the others?

Reply:
Because this shim is linked to the other projects (R4 and R4B). In this way we don't have to copy this shims to the other satellite projects.

Reply:
I should have known that, you showed me ;-)